src/main/kotlin/no/nav/familie/tilbake/config/OppdragMQConfig.kt

```diff
@@ -24,7 +25,8 @@ class OppdragMQConfig(@Value("\${oppdrag.mq.hostname}") val hostname: String,
                       @Value("\${oppdrag.mq.channel}") val channel: String,
```

Comment:
Her burde det vært 
@ConfigurationProperties("oppdrag.mq")
@ConstructorBinding
i stedet for @Configuration så slipper vi alle @value("")

Reply:
Det var allerede diskutert. Ref  (https://github.com/navikt/familie-tilbake/pull/57#discussion_r578455645)